types/react-flag-icon-css/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove anything apart from extends: dtslint/dt.json when writing new defintion

Reply:
I don't agree with this given _Common Mistakes_

> Formatting: Either use all tabs, or always use 4 spaces.